static/js/unread.js

```diff
@@ -667,6 +667,13 @@ export function get_missing_topics(opts) {
     return unread_topic_counter.get_missing_topics(opts);
 }
 
+export function get_thread_unread_count_from_message(msg) {
```

Comment:
Let's call this `message_to_conversation_unread_count(msg)`; I'd also prefer to have it live in `recent_topics_ui.js` rather than here, since this file tries to avoid dealing with full message objects.